docs/users/Shared_Configurations.mdx

```diff
@@ -67,14 +67,26 @@ module.exports = {
 
 If your project enables [typed linting](../getting-started/Typed_Linting.mdx), we suggest enabling the [`recommended-type-checked`](#recommended-type-checked) and [`stylistic-type-checked`](#stylistic-type-checked) configurations to start:
 
+> To use type-checking, you also need to configure `languageOptions.parserOptions` as shown in [typed linting docs](../getting-started/Typed_Linting.mdx).
+
 <Tabs groupId="eslint-config">
 <TabItem value="Flat Config">
 
 ```js title="eslint.config.mjs"
 export default defineConfig(
   eslint.configs.recommended,
+  // Added lines start
   tseslint.configs.recommendedTypeChecked,
   tseslint.configs.stylisticTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        projectService: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+  // Added lines end
 );
```

Comment:
This code was never meant to be complete, and by adding stuff that's already supposed to be there we suggest that it might be complete.

```suggestion
export default defineConfig(
  eslint.configs.recommended,
  // Added lines start
  tseslint.configs.recommendedTypeChecked,
  tseslint.configs.stylisticTypeChecked,
  // Added lines end
  // Other configuration...
);
```

This code is not teaching you how to enabled type-aware linting. It's assuming you already have the correct configuration and then showing you how to use shared config.

Reply:
This has been fixed